libs/upd/views/projects/src/lib/project-details/+state/projects-details.facade.ts

```diff
@@ -484,7 +484,7 @@ export class ProjectsDetailsFacade {
   ]).pipe(
     map(([data, lang]) => {
       const uxTests = data?.taskSuccessByUxTest.filter(
-        (uxTest) => uxTest.success_rate ?? false
+        (uxTest) => uxTest.success_rate !== undefined
```

Comment:
```suggestion
        (uxTest) => uxTest.success_rate || uxTest.success_rate === 0
```
This would cover all falsy values and explicitly accept 0, which is more robust and I think more clear.